lerna.json

```diff
@@ -2,15 +2,13 @@
   "lerna": "3.13.4",
   "npmClient": "npm",
   "packages": [
-    "examples/test",
```

Comment:
Why are these two removed here but the corresponding package.json files have been updated?

Reply:
these are not needed, shouldn't be here

Reply:
So should be delete the corresponding folders in examples?

Reply:
it was pushed accidently previously, it should never be added.